.github/sync-repo-settings.yaml

```diff
@@ -15,24 +15,13 @@ branchProtectionRules:
       - units (11)
       - cla/google
       - OwlBot Post Processor
-  - pattern: java7
-    isAdminEnforced: true
-    requiredApprovingReviewCount: 1
-    requiresCodeOwnerReviews: true
-    requiresStrictStatusChecks: false
-    requiredStatusCheckContexts:
-      - dependencies (8)
```

Comment:
I'm sure there is a reason, but don't we want to check all of these?

Reply:
These are for the java7 branch, which we don't have.

Reply:
Something like this: https://github.com/googleapis/java-pubsub/tree/java7
Some libraries need to maintain a long-term java7 support.

Reply:
Oh yes I see now. I missed the pattern main above.